### DISCUSSION TITLE
加载自定义 Iconfont 后图标加载错误

### DISCUSSION THREAD

**Discussion by Masterain98** [vuepress-theme-hope/vuepress-theme-hope#3532] [Q&A] [answered]
在 iconfont 中创建了一个新的项目并使用 css 引入到文档中，作为默认内置 iconfont 图标资源的拓展
```  
iconAssets: [
      "iconfont",
      "https://at.alicdn.com/t/c/font_3861247_26iz2fk3jhq.css"
  ],
```
同时将原本的图标引用方式进行了更改
```
#icon: license
icon: iconfont icon-license
```

但是在改动后，一部分图标出现了错误。以上面的 `license`为例，它原本的图标是这样的：
![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/10614984/911e1257-ed91-4402-b654-f2d6e7c82b2b)
在引入图标后，变成了这样：
![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/10614984/06260512-ced1-42e6-8ba6-ccb75aaa1cb2)

新的这个图标，在我的拓展图标集中，命名为 `faq`
![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/10614984/f489992c-8d73-4b64-9072-0bd078d1e7fa)

出现该问题不仅仅是这一个图标，还有其它的图标有这样的情况，均没有名称冲突。请问可能的原因是什么？

出现该问题的 PR： https://github.com/DGP-Studio/Snap.Hutao.Docs/pull/57

**Comment by Mister-Hope** [comment, marked as answer]
文档里有说这个问题，不同项目可能使用相同的私有字符，导致先后覆盖，所以最佳建议就是，你如果需要额外自定义图标，只使用一个自己的项目；
